perses/rjmc/geometry.py

```diff
@@ -719,13 +720,15 @@ def _corrected_reduced_potential(self, growth_system_generator, positions, platf
             #then the first one is the normal growth torsion force object and the second is the added torsion force object used to handle chirality and ring-closing constraints
             growth_system.removeForce(max(custom_torsion_forces))
 
-        mod_context = openmm.Context(growth_system, _integrator, openmm.Platform.getPlatformByName(platform_name))
+        from perses.dispersed.utils import configure_platform
+        platform = configure_platform(platform_name, fallback_platform_name='Reference', precision='double')
+        mod_context = openmm.Context(growth_system, _integrator, platform)
```

Comment:
maybe we should be using a context cache in `geometry.py`?  when i did my original pass through to fix rjmc, i wasn't familiar with `openmmtools.cache`, so i omitted, but maybe it's time to fix for good?  just for the sake of not having to specify a platform every time a context is created for bookkeeping purposes

Reply:
may have to create a `DEFAULT_PLATFORM` variable at the top of this file?